### PR TITLE
Add pvr.zattoo

### PR DESCRIPTION
**Pull request by rbuehlma**
zattoo.com is a Swiss TV provider also available in Germany, Spain, Denmark and UK.

The addon depends on this PR: https://github.com/xbmc/xbmc/pull/12728
I do already open this PR as I expect some changes due to the review.

@trummerjo: Is it ok with you, if I add this addon to the repository or would you prefer to do it yourself?

### CODE REVIEW

File: pvr.zattoo/pvr.zattoo.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+pvr.zattoo https://github.com/rbuehlma/pvr.zattoo 18.0.7
```
Comment (ksooo) [inline pvr.zattoo/pvr.zattoo.txt]: For the K18 development phase, replace the version tag by `master`, please.

We will pull the latest version from your master branch.

**Review by ksooo — CHANGES_REQUESTED** [review]

File: pvr.zattoo/pvr.zattoo.txt (hunk 1 above)
Comment (rbuehlma) [reply pvr.zattoo/pvr.zattoo.txt]: Done but please note that the current master is missing the libcurl removal

Comment (ksooo) [reply pvr.zattoo/pvr.zattoo.txt]: >  Done but please note that the current master is missing the libcurl removal

Then you should add it. Once this PR is merged, we pull whatever you have on master. It's your responsibility that there is something useful.  ;-)

Comment (rbuehlma) [reply pvr.zattoo/pvr.zattoo.txt]: I will, as soon as https://github.com/xbmc/xbmc/pull/12728 got merged. So please wait with merging this PR.

Comment (ksooo) [reply pvr.zattoo/pvr.zattoo.txt]: sure, absolutely no problem.

**Comment by rbuehlma** [comment]
Currently no more open issues in the addon. Also working on Android again. From my point of view, this PR can be merges as soon as you are happy with it.

**Review by ksooo — APPROVED** [review]

**Comment by ksooo** [comment]
Here we go. Welcome to the club. ;-)

**Pull request merged**